desktop/components/artwork_metadata_stub/test/index.js

```diff
@@ -0,0 +1,92 @@
+import cheerio from 'cheerio'
+import fs from 'fs'
+import jade from 'jade'
+import path from 'path'
+
+function render(locals) {
+  const filename = path.resolve(__dirname, '../index.jade')
+  return jade.compile(
+    fs.readFileSync(filename),
+    { filename: filename }
+  )(locals)
+}
```

Comment:
I just love that force's tests are getting less and less magical with every pass